src/pages/setting-page-client/change-password/ChangePassword.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from 'react';
+import { useForm, Controller, SubmitHandler } from 'react-hook-form';
+import * as Yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Checkbox } from 'antd';
+import { notification } from 'antd';
+import {
+	Form,
+	Input,
+	Button,
+	ControlStyle,
+	ErrorP,
+	InputBlock,
+	LabelBlock,
+	Title,
+	GlobalStyle,
+	Cancel,
+	ButtonBlock,
+} from './ChangePassword.styles';
+import { useTranslation } from 'react-i18next';
+import { usePasswordChangeMutation } from 'service/httpService';
+import { useAppSelector } from 'redux/hooks';
+import { RootState } from 'redux/store';
+
+export type FormPass = {
+	oldPassword: string;
+	newPassword: string;
+	confirmPassword: string;
+	email: string;
+};
+
+export interface Error {
+	data: {
+		message: string;
+	};
+}
+type Alert = 'success' | 'error';
+const schema = Yup.object({
+	oldPassword: Yup.string().required(),
+	newPassword: Yup.string()
+		.required()
+		.matches(/^(?=.{8,})/, 'Must Contain at least 8 digits')
+		.matches(/^(?=.*[0-9])/, 'Must Contain at least one Number')
+		.matches(/^(?=.*[!@#\$%\^&\*])/, 'Must Contain at least one Symbols'),
+	confirmPassword: Yup.string().required(),
+}).required();
+
+const ChangePassword = () => {
+	const [passwordShown, setPasswordShown] = useState<boolean>(false);
+	const [setPassword] = usePasswordChangeMutation();
+	const { user } = useAppSelector<RootState>(state => state);
+	const {
+		handleSubmit,
+		control,
+		reset,
+		formState: { errors },
+	} = useForm<FormPass>({
+		resolver: yupResolver(schema),
+	});
+
+	const { t } = useTranslation();
+
+	const alert = (type: Alert, message: string) => {
+		notification[type]({
+			message: message,
+		});
+	};
+
+	const onChange = (e: { target: { checked: boolean } }) => {
+		setPasswordShown(e.target.checked);
+	};
+
+	const onCancelButton = () => {
+		reset({ oldPassword: '', newPassword: '', confirmPassword: '' });
+	};
+
+	const onSubmit: SubmitHandler<FormPass> = async values => {
+		const { oldPassword, newPassword } = values;
+		if (values.newPassword !== values.confirmPassword) {
+			alert('error', 'Please check equality passwords');
+		} else {
+			try {
+				await setPassword({
+					oldPassword: oldPassword,
+					newPassword: newPassword,
+					email: user.email || '',
+				}).unwrap();
+				reset({ oldPassword: '', newPassword: '', confirmPassword: '' });
+				alert('success', 'Password changed successfully!');
+			} catch (e) {
+				reset({ oldPassword: '', newPassword: '', confirmPassword: '' });
```

Comment:
you duplicate { oldPassword: '', newPassword: '', confirmPassword: '' } very oten, move to constant please